jep/211/README.adoc

```diff
@@ -350,6 +350,23 @@ Non-goals in the specification are defined to limit the scope of work.
 The main objective is to get Jenkins running with Java 11,
 there will be follow-up tasks to cleanup Illegal Reflective Access warnings and to adopt new features.
 
+=== Support of Java 10 and 12
+
+Originally this JEP was targeting support of Java 10 and Java 11 early access.
+Java 10 is in the End of Life starting from 25.09.2018 when Java 11 was officially released.
+To limit the scope, Java 10 support was excluded from this JEP.
+We will not be testing new patches with Java 10 and providing Docker packages with this version.
```

Comment:
> will not be ... and providing Docker...

s/and/nor/

Reply:
good catch, and first not should be _neither_ then

> We will neither be testing new patches with Java 10 nor providing Docker packages with this version.

Reply:
```suggestion
We will neither be testing new patches with Java 10 nor providing Docker packages with this version.
```